java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture3/tasks/_1/Number1.java

```diff
@@ -0,0 +1,204 @@
+package com.zoolatech.lecture3.tasks._1;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/*
+Design classes that represent an orchestra, musicians and musical instruments (at least 3 different one).
+Each instrument has a name and a method to print a unique sound of the instrument.
+Each musician has a name and an associated instrument which he plays.
+It should be possible to distinguish two musicians.
+A musician should also have a method which should print his/her name and the name of an instrument.
+Then it needs to make the instrument sound. It should be possible to add/remove musicians from/to the orchestra.
+When the add/remove action is performed, the orchestra needs to print the action name + name of the musician.
+The orchestra should also have a method which commands all musicians to play the music.
+Inside of the main method, create a couple of instances of all classes and play the music.
+ */
+
+public class Number1 {
+
+    public static void main(String[] args) {
+        boolean exit = false;
+        String string;
+        Number1 number1 = new Number1();
+        Scanner scanner = new Scanner(System.in);
+        ArrayList<Musicians> list = new ArrayList<>();
+        number1.chooseInstrument(list);
+        Orchestra orchestra = new Orchestra(list);
+
+        System.out.println("Orchestra:");
+        System.out.println("1. Musician Info.\n2. Add Musician\n3. Remove Musician\n4. Play!\n5. Exit");
+        while (!exit) {
+            switch (scanner.nextInt()) {
+                case 1 -> {
+                    string = orchestra.toString();
+                    System.out.println(string);
+                    System.out.print("Amount of musicians:");
+                    orchestra.amountOfMusicians();
+                }
+                case 2 -> {
+                    number1.chooseInstrument(list);
+                }
+                case 3 -> {
+                    System.out.println("Choose witch one is removed: Starting from 0 to N-1");
+                    orchestra.removeMusician(scanner.nextInt());
+                }
+                case 4 -> {
+                    orchestra.play();
+                }
+                case 5 -> exit = true;
+                default -> System.out.println("ERROR");
+            }
+        }
+    }
+
+    public void chooseInstrument(ArrayList<Musicians> list) {
```

Comment:
The method can be changed a little bit: make it create a Musician instance and return it as the method result. Then, you could remove the input argument `list`, remove the new Orchestra instance creation, remove orchestra.addMusician code duplication, etc.